test/models/hook_test.rb

```diff
@@ -67,5 +67,10 @@ class HookTest < ActiveSupport::TestCase
     ensure
         Shipit.internal_hook_receivers = original_receivers
     end
+
+    test ".coerce_payload coerces anonymous user correctly" do
+      serialized = Hook.coerce_payload(lock_author: AnonymousUser.new)
+      assert_equal true, JSON.parse(serialized)["lock_author"]["anonymous"]
```

Comment:
Minor nitpick, but IIRC we have an `assert_json` test helper which allow this to be much cleaner.

Reply:
fixed.